pyproject.toml

```diff
@@ -0,0 +1,18 @@
+[tool.mypy]
+show_error_codes = true
+strict = true
+
+files = ["."]
+exclude = "setup.py"
+#mypy_path = "stubs"
+
+#[[tool.mypy.overrides]]
```

Comment:
Should this be removed?

Reply:
Yes, thanks. (This was copied from Sydent and I didn't go back and tidy this up!)

Reply:
And done in https://github.com/matrix-org/python-canonicaljson/pull/49/commits/2411f2f8d5dfe8767036d1aa99eb8a4677070631.